static/app/views/automations/components/dataConditionNodes.tsx

```diff
@@ -109,6 +134,9 @@ export const dataConditionNodesMap = new Map<DataConditionType, DataConditionNod
     {
       label: t('Event attribute'),
       dataCondition: <EventAttributeNode />,
+      details: (condition: DataCondition) => (
+        <EventAttributeDetails condition={condition} />
+      ),
```

Comment:
not getting a ton from this function that just calls the component
```suggestion
      detailsComponent: EventAttributeDetails,
```